OracleDatabase/SingleInstance/extensions/prebuiltdb/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # LICENSE UPL 1.0
 #
-# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2024 Oracle and/or its affiliates. All rights reserved.
```

Comment:
If this is a modified file, both the initial and this year's date should be shown.

Reply:
Fixed